package.json

```diff
@@ -5,13 +5,13 @@
   "main": "./build/index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "@elastic/elasticsearch": "7.13.0",
+    "@bugsnag/js": "7.20.0",
+    "@bugsnag/plugin-express": "7.19.0",
+    "@opensearch-project/opensearch": "2.2.0",
```

Comment:
Is there a types lib for opensearch? Should remove @types/elasticsearch and add that @ukrocks007

Reply:
I don't there is a types lib, types seems to included in the main package itself
https://github.com/opensearch-project/opensearch-js/blob/main/api/types.d.ts